packages/components/src/DescriptionList/DescriptionList.test.tsx

```diff
@@ -14,6 +14,21 @@ it("renders an object as a list of key value pairs", () => {
   expect(container).toMatchSnapshot();
 });
 
+it("allows duplicate terms", () => {
+  const logSpy = jest.spyOn(global.console, "error");
+
+  const { container } = render(
+    <DescriptionList
+      data={[
+        ["Issued", "2018-12-08"],
+        ["Issued", "2019-01-06"],
+      ]}
+    />,
+  );
+  expect(container).toMatchSnapshot();
+  expect(logSpy).toHaveBeenCalledTimes(0);
```

Comment:
Hi @darryltec. Do you mean that I should do something like?:
`  expect(logSpy).toHaveBeenCalledWith(
    'Warning: Encountered two children with the same key, `%s`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.%s", "Issued", ',
  );`
That would be more specific but also doesn't seem great as I expect that warning message might change at some point. Would be an issue for an upgrade, though.

Sorry, still ramping up my React skills and not sure the best practice for testing these kind of warnings.

Reply:
we could just check for partial text like `Warning: Encountered two children with the same key` I'd rather be specific and it failing on upgrade than it being very generic and failing randomly. Especially locally on dev as we have some pre-commit hooks that would fail.